docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

```diff
@@ -33,15 +33,20 @@ If you're using an older version of NorthstarProton, replace the information abo
 
 ## Lutris (Wine)
 
-1. If you don't already have the game downloaded, install the game [from here.](https://lutris.net/games/titanfall-2/)
-2. Download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
-3. Extract all contents of the file to your Titanfall 2 folder
-4. **If you have the game installed on Lutris:** right click _Titanfall 2_ > _Configure_ > _Game Options_ > Set _Executable path_ to _NorthstarLauncher.exe_
-5. **Otherwise:** click the `+` button in the top left > set the name to whatever and _Runner_ to _Wine_ > click on _Game options_ > set _Executable path_ to _NorthstarLauncher.exe_ then save.
+1. Install the EA App first [from here.](https://lutris.net/games/ea-app/)
+2. Download and install Titanfall 2 from the EA App, run the vanilla version at least once.
+3. Download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page.
+4. Extract all contents of the file to your Titanfall 2 folder.
+4. **If you have the game installed on Lutris:** right click _EA App_ > _Duplicate_ > _Yes_. Then right click the new _EA App 2_ > _Configure_ > _Game Options_ > Set _Executable path_ to _NorthstarLauncher.exe_ and add `-northstar` to the _Arguments_. Optionally, change the name of the EA App 2 shortcut to Northstar and change the identifier to `titanfall-2`
```

Comment:
Is there a reason you say to run `NorthstarLauncher.exe` and then still use `-northstar`? `-northstar` is just used when running vanilla with the modified wsock32 library to redirect the call to Northstar instead.

Reply:
My mistake, once I finally got the game to launch, that was the most recent thing I tried to get it to work. I removed the launch option and it still launches Northstar successfully now.